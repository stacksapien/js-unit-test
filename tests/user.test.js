// tests/user.test.js

const getUser = require("../src/user");

describe("User Module", () => {
  test("returns the correct user object", () => {
    const user = getUser();
    expect(user).toMatchSnapshot();
  });
});
