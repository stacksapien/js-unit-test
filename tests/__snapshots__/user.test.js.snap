// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Module returns the correct user object 1`] = `
{
  "email": "john.doe@example.com",
  "id": 1,
  "name": "John Doe",
}
`;
